run-under-qemu.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Invocation run_under_qemu.sh [qemu_path] [test_name] [test_args...]
+
+if [ $# -lt 2 ]; then
+    echo "Usage: run_under_qemu.sh [qemu_path] [test_name] [test_args...]"
```

Comment:
Can we make this more generic, by allowing some of the qemu-args to be passed in as well so that it can work for more targets? For one, the ability to parametrize `-cpu' would be useful in testing different configurations.